functions/src/events.ts

```diff
@@ -248,3 +249,49 @@ export const addInstagramPoints = functions.https.onCall(async (data, context) =
     return { success: true };
 });
 
+
+export const eventLogDelete = functions.https.onCall(async (data, context) => {
+    const uid = data.uid || context.auth?.uid;
```

Comment:
This is a MASSIVE security vulnerability. If any user is able to call this function, they can delete any event log they want. I've just realized that `eventSignIn` and `eventSignOut` have this vulnerability as well and will be refactoring those as soon as possible. Do *not* deploy this cloud function

Reply:
okay. undeployed the function.